client/jetpack-connect/woo-loader-stages.tsx

```diff
@@ -25,12 +23,14 @@ export const CreatingYourAccountStage = {
 	progress: 70,
 };
 
-export const OpeningTheDoorsStage = {
-	title: __( 'Opening the doors' ),
-	image: OpeningImage,
+// This is a placeholder stage that is used so the progress bar can reach 100%
+export const PlaceholderStage = {
+	title: __( 'Creating your account' ),
```

Comment:
Since we’re using the `ConnectingYourAccount` as the first stage, I think it makes sense to update the title to ‘Connecting your account’ for consistency.
```suggestion
	title: __( 'Connecting your account' ),
```

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/104322/commits/91954428d5b4bd1fa7feb101ae39459ea0a8094a